### ISSUE TITLE
Z_DATA_ERROR

### ISSUE THREAD

**Issue by i-am-henri** [nfriedly/node-unblocker#243]
Hey, I tried the npm package, and receave now this error:
![image](https://github.com/nfriedly/node-unblocker/assets/98414850/31e909d5-97cb-43e2-9a13-b94b711c345d)

My **node** Version: 18.18.2
I tried this out with nodemon and without nodemon.

**Comment by i-am-henri** [comment]
Code:
![image](https://github.com/nfriedly/node-unblocker/assets/98414850/1553b795-352e-4d13-aca4-45882576e712)

**Comment by nfriedly** [comment]
Hum, it looks like that site supports [br encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding#br), and node-unblocker does not; that might be the source of the issue.

**Comment by i-am-henri** [comment]
ok, is there a way to fix that? (thanks for the fast reply)

**Comment by nfriedly** [comment]
Yeah, I just tested it out, and it's a CSS file that's claiming to be gzip encoded, but my suspicion is that it actually isn't, I think it's brotili (br) with an incorrect header. 

It looks like node.js now has built-in broti support, so it shouldn't be too much trouble to add to the proxy.

**Comment by i-am-henri** [comment]
do you have an example code piece, I'm very new to this package, and don't know how ti add broti to the proxy.

**Comment by nfriedly** [comment]
So, I played with brotili a bit and it doesn't seem to solve the problem. However I noticed that if I visit my local server over https (via nginx proxy manager), then everything works, but if I switch back to http then it crashes again. I'm not really sure why that is...

**Comment by nfriedly** [comment]
I also noticed that some of the redirects the server sends out use http, which causes them to be refused by the browser, but that's an unrelated bug.
